recipes/pybind_json/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
```

Comment:
I would recommend to create a build folder and let the cmake files there, something like
```suggestion
mkdir -p build
pushd build
```
Because if in the future you need to investigate any problem the whole ``cmake`` files and build files will be at that folder